### PR TITLE
i18n(zh-cn): Update configuration-reference.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update configuration-reference.mdx

#### Related issues & labels (optional)

- #6437

### CODE REVIEW

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -1127,3 +1153,68 @@ Astro 提供了实验性标志，以便用户提前使用新功能。这些标
 	},
 }
 ```
+
+### experimental.clientPrerender
+
+<p>
+
+**Type:** `boolean`<br />
```
[the remark this answers is not in the document]
Comment (huyikai) [reply src/content/docs/zh-cn/reference/configuration-reference.mdx]: 已更新

[the remark this answers is not in the document]
Comment (liruifengv) [reply src/content/docs/zh-cn/reference/configuration-reference.mdx]: 点一下 resolve conversations

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 2]
```diff
@@ -1077,6 +1077,32 @@ export default defineConfig({
 包括默认语言在内所有的 URL 都会是 `example.com/[locale]/content/`。
 包括默认语言的所有语言都会使用本地化的文件夹。
 
+#### i18n.routing.redirectToDefaultLocale
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1080]: 上面的 `i18n.routing.prefixDefaultLocale` 也应该是4个 `#`，一块改一下

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🚀

**Pull request merged**